public/pages/CreateMonitor/components/VisualGraph/VisualGraph.js

```diff
@@ -5,133 +5,106 @@
-    const { bucketValue, bucketUnitOfTime, groupBy, aggregations } = values;
-    const aggregationType = aggregations[0]?.aggregationType;
-    const fieldName = aggregations[0]?.fieldName;
     const { hint } = this.state;
 
-    const xDomain = getXDomain(data);
-    const yDomain = getYDomain(data);
-    const annotations = getAnnotationData(xDomain, yDomain, thresholdValue);
-
-    const xTitle = values.timeField;
-    const yTitle = getYTitle(values);
-    const leftPadding = getLeftPadding(yDomain);
-    const markData = getMarkData(data);
-    const title = aggregationType
-      ? `${aggregationType?.toUpperCase()} OF ${fieldName}`
-      : 'COUNT of documents';
-    const description = getGraphDescription(bucketValue, bucketUnitOfTime, groupBy);
-
-    return (
-      <ContentPanel
-        title={title}
-        titleSize="s"
-        panelStyles={{ paddingLeft: '10px' }}
-        description={description}
-      >
-        <FlexibleXYPlot
-          height={400}
-          xType="time"
-          margin={{ top: 20, right: 20, bottom: 70, left: leftPadding }}
-          xDomain={xDomain}
-          yDomain={yDomain}
-          onMouseLeave={this.resetHint}
-        >
-          <XAxis title={xTitle} />
-          <YAxis title={yTitle} tickFormat={formatYAxisTick} />
-          <LineSeries data={data} style={LINE_STYLES} />
-          <MarkSeries data={markData} sizeRange={SIZE_RANGE} onNearestX={this.onNearestX} />
-          {annotation && <LineSeries data={annotations} style={ANNOTATION_STYLES} />}
-          {hint && (
-            <Hint value={hint}>
-              <div style={HINT_STYLES}>({hint.y.toLocaleString()})</div>
-            </Hint>
-          )}
-        </FlexibleXYPlot>
-      </ContentPanel>
-    );
-  };
-
-  renderAggregationXYPlot = (data, groupedData) => {
-    const { annotation, thresholdValue, values, fieldName, aggregationType } = this.props;
-    const { hint } = this.state;
     const xDomain = getBufferedXDomain(data, values);
     const yDomain = getYDomain(data);
     const annotations = getAnnotationData(xDomain, yDomain, thresholdValue);
-    const xTitle = values.timeField;
-    const yTitle = fieldName;
     const leftPadding = getLeftPadding(yDomain);
-    const width = computeBarWidth(xDomain);
 
-    const title = aggregationType
-      ? `${aggregationType?.toUpperCase()} OF ${fieldName}`
-      : 'COUNT of documents';
-    const legends = groupedData.map((dataSeries) => dataSeries.key);
+    // reasonable defaults for PPL-only graph
+    const xTitle = values?.timeField || 'Time';
+    const yTitle = '';
+
+    // width for single-series bars
+    let width = computeBarWidth(xDomain);
+    if (!Number.isFinite(width) || width <= 0) {
+      // fallback: ~1 hour in ms
+      width = 60 * 60 * 1000;
+    }
+    const rectData = getRectData(data, width, 0, 1);
 
     return (
       <ContentPanel
-        title={title}
+        title="Results"
```

Comment:
The VisualGraph is still being used by the alertingV1 UI, right? Should the title be changed for non-v2 monitors? 

Has the visual graph been tested with these changes to confirm it still functions properly for v1 monitors?